Automattic/jetpack
# VideoPress: add query object to the getVideos() selector

This PR adds a query object parameter to the getVideos() selector to be able to define the search criteria when requesting the video data. This PR improves the doc according to these changes.

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: add query object to the getVideos() selector

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to VideoPress admin page
* Use Redux Dev Tool: [Firefox](https://addons.mozilla.org/en/firefox/addon/reduxdevtools/) | [Chrome](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en)
* Open the dev console

* confirm that the it doesn't request when the query doesn't change

https://user-images.githubusercontent.com/77539/188757980-5a64cd79-534a-405c-961a-00098d40e223.mov
